### PR TITLE
Ndbit6

### PR DESCRIPTION
**Pull request by ndGarage**
Board ndGarage bit6

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -121,6 +121,7 @@ jobs:
         - "metro_nrf52840_express"
         - "mini_sam_m4"
         - "monster_m4sk"
+        - "ndbit6"
```
Comment (ladyada) [inline .github/workflows/build.yml]: wanna name this ndgarage_ndbit6?

**Comment by ladyada** [comment]
woops can u rebase? we had another board come in the meantime :)

**Comment by ndGarage** [comment]
My bad. I will create a new PR soon.

**Comment by ladyada** [comment]
ok i tried to fix with github - might be ok now!

**Comment by ladyada** [comment]
done!

**Pull request merged**